docker/cassandra/entry-point

```diff
@@ -12,4 +12,4 @@ do
 done
 )
 
-/usr/sbin/cassandra
\ No newline at end of file
+exec /usr/sbin/cassandra
```

Comment:
`exec` to skip child processes and so exit works properly.

Reply:
Just so I'm clear, what was wrong previously?

Reply:
`cassandra` becomes a child of `bash`, which is unnecessary, and IIRC can cause problems with things like `SIGTERM` handling. In general, unless the parent needs to be kept around for some reason, `exec` is better.